pdm/release/action.yml

```diff
@@ -0,0 +1,117 @@
+name: Release
+description: Bump version and publish release assets
+
+inputs:
+  kind:
+    description: Kind of project to release (lib/app)
+    required: true
+    default: app
+  pypi-token:
+    description: a Token to Ledger private PyPI
+    required: true
+    default: ""
+  github-token:
+    description: a Github token with 
+    required: true
+  increment:
+    description: "Kind of increment (optional: MAJOR|MINOR|PATCH)"
+    default: ''
+    required: false
+
+
+outputs:
+  url: 
+    description: The generated Github Release URL
+    value: ${{ steps.release.outputs.url }}
+  version: 
+    description: The released version
+    value: ${{ steps.bump.outputs.version }}
+  documentation: 
+    description: The released documentation URL
+    value: ${{ steps.doc.outputs.url }}
+
+
+runs:
+  using: composite
+  steps:
+    - name: Clone
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        token: ${{ inputs.github-token }}
+
+    - name: Bump using commitizen
+      id: bump
+      uses: commitizen-tools/commitizen-action@master
+      with:
+        github_token: ${{ inputs.github-token }}
+        changelog_increment_filename: body.md
+        extra_requirements: emotional
+        increment: ${{ inputs.increment }}
+        debug: true
+
+    - name: Set up Python and PDM
+      uses: pdm-project/setup-pdm@v3
+      if: inputs.kind == 'lib'
+      with:
+        python-version: "3.11"
+        cache: true
+
+    # Build once to publish the same package on every repository
+    - name: Build distribution
+      if: inputs.kind == 'lib'
+      run: pdm build
+
+    - name: Push to GemFury
+      if: inputs.kind == 'lib' && env.PDM_PUBLISH_USERNAME != null
+      env:
+        PDM_PUBLISH_REPO: https://push.fury.io/ledger
+        PDM_PUBLISH_USERNAME: ${{ inputs.pypi-token }}
+        PDM_PUBLISH_PASSWORD: unused # tricks github actions YAML parser and PDM test for empty password
+      run: pdm publish --no-build
+
+    # Disabled until Nexus is confirmed working
+    # - name: Push to our internal Nexus
+    #   if: inputs.kind == 'lib' && env.PDM_PUBLISH_USERNAME != null && env.PDM_PUBLISH_PASSWORD != null
+    #   env:
+    #     PDM_PUBLISH_REPO: https://{% raw %}${{ secrets.GREEN_NEXUS_HOST }}{% endraw %}/repository/pypi-internal/
+    #     PDM_PUBLISH_USERNAME: {% raw %}${{ secrets.GREEN_NEXUS_USER }}{% endraw %}
+    #     PDM_PUBLISH_PASSWORD: {% raw %}${{ secrets.GREEN_NEXUS_PASSWORD }}{% endraw %}
+    #   run: pdm publish --no-build
```

Comment:
It's working and used by at least a project so we should keep it enabled for publication of libraries.

Reply:
Glad to hear it because I have been asking for the past year and each time I was told "it's not working" and then "it's not the target anymore, we are focusing on artifactory pro".
I'll be glad to enable it 🎉 (but I'll need access to test it's working)

Reply:
Well I know that at least on project uses it so it better fills it.
However we might drop it to artifactory in the near future for multiple reasons but the discussions are ongoing.
In the meanwhile we want to keep this activated :)

Reply:
OK, I uncommented it, fixed it (but I haven't tested it yet)